src/interfaces.js

```diff
@@ -132,5 +132,5 @@ interface ConfigureRouter {
   /**
   * Implement this hook if you want to add sub-routes to your view.
   */
-  configureRouter(config: RouterConfiguration, router: Router);
+  configureRouter(config: RouterConfiguration, router: Router): any;
```

Comment:
@StrahilKazlachev, I'm wondering, how did You come up with the return type of configureRouter method?
(By the way, it is the only method in [my sample](https://gist.github.com/atsu85/2777454d6ee215a1b9d3911ce157828d#file-viewhooks-d-ts-L19-L24) that isn't written by myself, but my teammate). 
The [official documentation](http://aurelia.io/hub.html#/doc/article/aurelia/framework/latest/cheat-sheet/6) has one place where the signature is: 
`configureRouter(config: RouterConfiguration, router: Router): void`
and five places where the second argument is omitted from the signature:
`configureRouter(config: RouterConfiguration): void`
but none of them has anything else but `void` as the return type.

From the source i found 2 places, that use that method:
* [route-loading.js#L109](https://github.com/aurelia/router/blob/34119d1ce827f10fe5d3c7c97133d83ad922eab6/src/route-loading.js#L109)
* [app-router.js#L66](https://github.com/aurelia/router/blob/34119d1ce827f10fe5d3c7c97133d83ad922eab6/src/app-router.js#L66)
but i didn't spend any time to figure out what are the potential return types. 

@EisenbergEffect, @bryanrsmith - could You clarify the return type of `configureRouter`?